cypress/integration/seededFlows/podcastFlows/togglePodcastPlayback.spec.js

```diff
@@ -0,0 +1,18 @@
+describe('Toggle Podcast playback', () => {
+  beforeEach(() => {
+    cy.testSetup();
+  });
+
+  it('should toggle podcast playback', () => {
+    cy.visit('/pod');
+    cy.contains('div', 'Example media | crow call').click();
```

Comment:
I notice the link doesn't have an accessible name, so you had to workaround by targeting the `div` 🙃  I don't think any change to that link falls under the scope of your PR, but just to say I'll make a note and we can get that fixed in future - nice work uncovering the issue with the test!